### ISSUE TITLE
In 4.6.3.1 assets can no longer be compiled

### ISSUE THREAD

**Issue by mkon** [ai/fotoramajs#7]
I am getting this error
```
Sass::SyntaxError: Invalid CSS after "....png) no-repeat": expected "{", was "}@media (-webki..."
  (in /var/rails/hkdemo/vendor/bundle/ruby/2.1.0/gems/fotoramajs-4.6.3.1/vendor/assets/stylesheets/fotorama.css.scss:5)
```

It seems previous versions had a proper scss stylesheet, since 4.6.3.1 it is an already compressed css file that can no longer be processed by the sass compiler. I rather not include it as a separate stylesheet to circumvent the assets pipeline.

**Comment by ai** [comment]
Sorry again. Fixed 02e732a67bbbcd6b7a7682af7b3ff6d0b183870f

**Comment by ai** [comment]
Released in 4.6.3.2.
